_layouts/default.html

```diff
@@ -21,47 +37,55 @@
         <!-- To automatically render math in text elements, include the auto-render extension: -->
         <script defer src="https://cdn.jsdelivr.net/npm/katex@0.15.2/dist/contrib/auto-render.min.js" integrity="sha384-+XBljXPPiv+OzfbB3cVmLHf4hdUFHlWNZN5spNQ7rmHTXpd7WvJum6fIACpNNfIR" crossorigin="anonymous"
             onload="renderMathInElement(document.body);"></script>
-	</head>
-	<body>
-		<header>
-			<h1><a href="/">{{ site.title }}</a></h1>
-			<span>{{ site.tagline }}</span>
-		</header>
+    </head>
+    <body>
+        <header>
+            <h1><a href="/">{{ site.title }}</a></h1>
+            <span>{{ site.tagline }}</span>
+        </header>
+
+        <div class="hfeed">
+            {{ content }}
+        </div>
+        
+        <aside>
+            <div id="drawer">
+                <div id=drawer-contents>
+                    <nav>
+                        <a href="/archive.html">Archive</a>
+                        <a href="/about.html">About me</a>
+                        <a href="/projects.html">Interesting Projects</a>
+                        <a href="/tags.html">Tags</a>
+                        <a target="_blank" href="/feed.xml">RSS Feed</a>
+                    </nav>
+
+                    <h4  style="margin-top: 1.5em;">Elsewhere on the internet...</h4>
+                    <ul>
+                            <!--<li><a href="http://eepurl.com/iKGOdY">Mailinglist</a></li>-->
+                            <li><a href="http://twitter.com/pepijndevos" target="_blank">Twitter</a></li>
+                            <li><a href="http://github.com/pepijndevos" target="_blank">Github</a></li>
+                    </ul>
+
+                    {% if site.related_posts %}
+                    <h4 style="margin-top: 1.5em;">Remotely related posts</h4>
```

Comment:
Really don't like these, especially the ones that are clearly not one-of but repeat several times.

Reply:
they are repeated exactly two times lmao